src/views/preremplissage.vue

```diff
@@ -0,0 +1,324 @@
+<script setup lang="ts">
+import LoadingModal from "@/components/loading-modal.vue"
+import InputDate from "@/components/input-date.vue"
+import InputDepcom from "@/components/input-depcom.vue"
+import { ref, computed, watch, nextTick } from "vue"
+import { useStore } from "@/stores/index.js"
+import { Answer } from "@lib/types/store.d.js"
+import axios from "axios"
+import * as Sentry from "@sentry/vue"
+
+const store = useStore()
+
+const family_name = ref<string | undefined>()
+const given_names = ref<string | undefined>()
+const birthdate = ref<string | undefined>()
+const postcode = ref<string | undefined>()
+const gender = ref<string | undefined>()
+const email = ref<string | undefined>()
+const phone = ref<string | undefined>()
+const cityName = ref<string | undefined>()
+const updating = ref<boolean>(false)
+const formError = ref<boolean>(false)
+const prefillSuccess = ref<boolean>(false)
+const prefillSuccessAlert = ref<HTMLElement | null>(null)
+
+watch(prefillSuccess, (newValue) => {
+  if (newValue === true) {
+    nextTick(() => {
+      if (prefillSuccessAlert.value instanceof HTMLElement) {
```

Comment:
Pourquoi ne pas scroll to top ? ça enlève de la logique et je trouve le comportement un peu plus normal, t'en pense quoi ? Là on arrive sur ça : 
<img width="1693" alt="image" src="https://github.com/betagouv/aides-jeunes/assets/4059803/fb08f105-6253-4e86-902f-3f3f2d2daea2">

Reply:
Oui, c'est aussi une solution ! Je vais modifer